draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.scss

```diff
@@ -19,21 +18,47 @@ $dt-tooltip-content-border: $color-gray-300;
 
 .tooltipContent {
   border-radius: $border-solid-border-radius;
-  transition: opacity $ca-duration-fast, transform $ca-duration-fast;
-  padding: $ca-grid / 2 $ca-grid / 2;
-  border: $border-solid-border-width $border-solid-border-style
-    $dt-tooltip-content-border;
+  transition: opacity $animation-duration-fast,
+    transform $animation-duration-fast;
+  padding: $spacing-sm $spacing-sm;
   box-shadow: $shadow-small-box-shadow;
   text-align: center;
   max-width: 400px;
-  // I'm having a problem getting the tooltip width to match the size of the text content.
-  // As a workaround, I'm just going to set a min width of 200px. It should be enough
-  // in most cases.
-  min-width: 200px;
-  @include kz-typography-paragraph-body($size: 14);
-  top: initial; // Overrides the `top` value of kz-typography-paragraph-body
   color: $color-purple-800;
-  background-color: white;
+  font-family: $typography-paragraph-body-font-family;
+  font-size: $typography-heading-6-font-size;
+  font-weight: $typography-paragraph-body-font-weight;
+  $letter-spacing: $typography-paragraph-body-letter-spacing;
```

Comment:
Should there be a `$` at the start here?

Reply:
Removed now, thanks!